examples/get_started_yolo.py

```diff
@@ -0,0 +1,209 @@
+        'tennis racket', 'bottle', 'wine glass', 'cup', 'fork', 'knife', 'spoon', 'bowl', 'banana', 'apple',
+        'sandwich', 'orange', 'broccoli', 'carrot', 'hot dog', 'pizza', 'donut', 'cake', 'chair', 'couch',
+        'potted plant', 'bed', 'dining table', 'toilet', 'tv', 'laptop', 'mouse', 'remote', 'keyboard', 
+        'cell phone', 'microwave', 'oven', 'toaster', 'sink', 'refrigerator', 'book', 'clock', 'vase', 'scissors', 
+        'teddy bear', 'hair drier', 'toothbrush']
+
+def extract_predictions(predictions_, conf_thresh):
+    # Get the predicted class
+    predictions_class = [COCO_INSTANCE_CATEGORY_NAMES[i] for i in list(predictions_["labels"])]
+    #  print("\npredicted classes:", predictions_class)
+    if len(predictions_class) < 1:
+        return [], [], []
+    # Get the predicted bounding boxes
+    predictions_boxes = [[(i[0], i[1]), (i[2], i[3])] for i in list(predictions_["boxes"])]
+
+    # Get the predicted prediction score
+    predictions_score = list(predictions_["scores"])
+    # print("predicted score:", predictions_score)
+
+    # Get a list of index with score greater than threshold
+    threshold = conf_thresh
+    predictions_t = [predictions_score.index(x) for x in predictions_score if x > threshold]
+    if len(predictions_t) > 0:
+        predictions_t = predictions_t  # [-1] #indices where score over threshold
+    else:
+        # no predictions esxceeding threshold
+        return [], [], []
+    # predictions in score order
+    predictions_boxes = [predictions_boxes[i] for i in predictions_t]
+    predictions_class = [predictions_class[i] for i in predictions_t]
+    predictions_scores = [predictions_score[i] for i in predictions_t]
+    return predictions_class, predictions_boxes, predictions_scores
+
+
+def plot_image_with_boxes(img, boxes, pred_cls, title):
+    plt.style.use('ggplot')
+    text_size = 1
+    text_th = 3
+    rect_th = 1
+
+    for i in range(len(boxes)):
+        cv2.rectangle(img, (int(boxes[i][0][0]), int(boxes[i][0][1])), (int(boxes[i][1][0]), int(boxes[i][1][1])),
+                      color=(0, 255, 0), thickness=rect_th)
+        # Write the prediction class
+        cv2.putText(img, pred_cls[i], (int(boxes[i][0][0]), int(boxes[i][0][1])), cv2.FONT_HERSHEY_SIMPLEX, text_size,
+                    (0, 255, 0), thickness=text_th)
+
+    plt.figure()
+    plt.axis("off")
+    plt.title(title)
+    plt.imshow(img.astype(np.uint8), interpolation="nearest")
+    plt.show()
+    
+'''
+#################        Evasion settings        #################
+'''
+eps = 32
+eps_step = 2
+max_iter = 10
+
+
+'''
+#################        Model definition        #################
+'''
+MODEL = 'yolov3' # OR yolov5
+
+
+if MODEL == 'yolov3':
+
+    from pytorchyolo.utils.loss import compute_loss
+    from pytorchyolo.models import load_model
+
+    class Yolo(torch.nn.Module):
```

Comment:
Could you please update the docstrings of `PyTorchYolo` where the model is input to mention that a wrapper is required as shown in this example?